java/build.gradle.kts

```diff
@@ -1,29 +1,23 @@
-import org.gradle.api.publish.maven.internal.publication.DefaultMavenPublication
-
 plugins {
     `java-library`
     `maven-publish`
-    id("com.google.protobuf") version "0.9.4"
+    id("dev.sigstore.sign") version "0.8.0"
     id("com.diffplug.spotless") version "6.25.0"
+    `signing`
 }
 
-description = "Code generated library for the Sigstore bundle format protobufs"
-
-sourceSets {
-    main {
-        proto {
-            srcDir("../protos/")
-        }
-    }
-}
+description = "Sigstore protobuf spec protos bundled into a jar"
```

Comment:
we no longer generate code, that's delegated to the consumer (easier to handle version conflicts)